components/style/properties/longhand/box.mako.rs

```diff
@@ -353,6 +353,7 @@ ${helpers.single_keyword("overflow-clip-box", "padding-box content-box",
 
 // FIXME(pcwalton, #2742): Implement scrolling for `scroll` and `auto`.
 ${helpers.single_keyword("overflow-x", "visible hidden scroll auto",
+                         extra_gecko_values="clip",
```

Comment:
😄  Btw, I needed to update overflow-{x,y} to `clip` if content is `paint` in `apply_declarations` function.